FullyNoded/Helpers/CoreDataService.swift

```diff
@@ -37,6 +37,7 @@ class CoreDataService {
         }
     }
     
+    // TODO: Create a data type to simplify the use of [[String:Any]]
```

Comment:
I use Structs for most of the dicts that can be returned by Core Data: https://github.com/Fonta1n3/FullyNoded/tree/master/FullyNoded/Structs

Am all ears about ways to improve it